src/app/pages/animal/register-community-animal-by-census/register-community-animal-by-census.page.html

```diff
@@ -0,0 +1,221 @@
+<app-general-container-component>
+  <h1>Registrar animales comunitarios</h1>
+
+  <mat-form-field appearance="outline" class="w-100">
+    <mat-label>Buscar por cédula</mat-label>
+    <input matInput [formControl]="cedulaControl" />
+  </mat-form-field>
+
+  <button mat-raised-button color="primary" (click)="onSearchByCedula()">
+    Buscar propietario
+  </button>
+</app-general-container-component>
+
+<ng-template #dialogConfirmarRegistroUsuario>
+  <h2 mat-dialog-title>¿Registrar nuevo propietario?</h2>
+  <mat-dialog-content>
+    No se encontró un propietario con esta cédula. ¿Desea registrar uno nuevo?
+  </mat-dialog-content>
+  <mat-dialog-actions align="end">
+    <button mat-button mat-dialog-close="false">Cancelar</button>
+    <button mat-button mat-dialog-close="true" color="primary">Registrar nuevo propietario</button>
+  </mat-dialog-actions>
+</ng-template>
+
+<ng-container *ngIf="userExists()">
```

Comment:
cambiar por sintaxis moderna de 

@if

```suggestion
@if(userExists()) {
   <ng-container>
}
```

Reply:
Done